### ISSUE TITLE
Characters not entering stage sometimes in Tag Team 

### ISSUE THREAD

**Issue by AlekBunny** [ikemen-engine/Ikemen-GO#1043]
### Describe the bug

I'm using Ikemen Go 0.99.0-rc.1 Sometimes while playing Tag Team 3x3 or 4x4 the player or AI char doesn't enter the stage after a team member K.O. It happened with the default Kung-Fu Man and also with some Mugen chars. The fist time it occured I thought it could be some conflict with Mugen chars but then testing out with the default Kung-Fu Man the bug showed again.

### To Reproduce

I've downloaded chars from mugenarchive.com but the bug also happens with default Kung-Fu Man.

Start Ikemen Go > Arcade > Team Arcade > Select Tag 3x3 or 4x4 and play. In Watch Mode (AI v.s AI) the bug can be reproduced too.
Start Ikemen Go > Watch Mode > CPU Match > Select Tag 3x3 or 4x4 and watch.

### Expected behavior

I expect the team members to enter the stage to continue fighting after some member K.O.

### Screenshots / Video

https://user-images.githubusercontent.com/69438957/224516194-49303994-9adb-4e19-ace3-05ba642be87a.mp4

### Engine Version (or source code date)

v0.99.0-rc.1

### Operating system

Windows

**Comment by potsmugen** [comment]
I think this one's an issue with the function that checks which character to tag in (fTagGetPartnerId). I ran 30 matches with the new function code, and 30 with the old code. The new code caused the bug 5 times while the old one did not cause it at all.

**Comment by AlekBunny** [comment]
@potsmugen Hi! Thanks for the info! =:) What Ikemen version did you use? The one that isn't causing the this bug?

**Comment by potsmugen** [comment]
I used the same one you did, then modified it a bit to see what was causing the bug. Until it gets fixed you can just use the tag.zss file from the 0.98.2 version.

**Comment by AxeLanderMoreira** [comment]
Jumping into the discussion, I was testing the same scenario (watch mode tag team 4v4) to try and reproduce an old crash (#658 ), which by the way has been fixed, and got the same issue described here.

@potsmugen , I tried using just the tag.zss file from 0.98.2 version*, while keeping all of the other binary files and data files from 0.99.1 version; then I get a different bug where the other characters in the team will not leave the screen upon the beginning of the round. So I guess that the old tag.zss file is not really compatible.

* more specifically, I have downloaded it from here:
https://raw.githubusercontent.com/ikemen-engine/Ikemen-GO/v0.98.2/data/tag.zss
